src/website/app/demos/Button/examples/element.js

```diff
@@ -0,0 +1,22 @@
+/* @flow */
```

Comment:
This could possibly just be a unit spec that compares snapshots, rather than a demo example. Not a big deal, either way.

Reply:
Agreed.  There is no need for this file as we never need to inspect it visually.  It can all be handled via a simple snapshot test.